CommandLine/protos/game.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto2";
+package buffers;
+
+import "resources/Sprite.proto";
+import "resources/Sound.proto";
+import "resources/Background.proto";
+import "resources/Object.proto";
+import "resources/Script.proto";
+import "resources/Shader.proto";
+import "resources/Path.proto";
+import "resources/Font.proto";
+import "resources/Timeline.proto";
+import "resources/Room.proto";
+
+message Project {
```

Comment:
This has to be the most amazing thing about this change.

Reply:
I don't get what you mean. The fact that it's a domain specific language? The fact that it's easy for a user to add a custom resource type? The fact that it can be changed in backwards-compatible ways? The fact that accessing unset properties doesn't segfault? The fact it doesn't need to look for Java?